### PR TITLE
Fix old dockercfg secret

### PR DESCRIPTION
**Pull request by lewismarshall**

### CODE REVIEW

**Comment by lewismarshall** [comment]
@jaykeshur @timgent

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (vaijab) [inline .gitignore:1]: No new line :octopus:

**Comment by timgent** [comment]
LGTM. Though couldn't we just give the secret that name to start with rather than having to move it here?

**Comment by lewismarshall** [comment]
@timgent we could but the name was renamed during an old commit and we have 6 or 8 or so service lines using this now...

**Pull request merged**